src/lib/UXDController.ts

```diff
@@ -68,82 +69,82 @@ export class UXDController {
 
   public mint({
     amount,
-    slippage,
+    targetPrice,
     signer,
     collateral,
   }: {
     amount: number;
-    slippage: number;
+    targetPrice: number;
```

Comment:
is targetPrice just another name for slippage, or is it something we need to compute with a specific formula?

Reply:
It's the price limit matching the slippage UI setting, which the frontend has to calculate, ideally through the client